lib/galaxy/managers/api_keys.py

```diff
@@ -1,26 +1,40 @@
+from typing import List
+
+from galaxy.model import (
+    APIKeys,
+    User,
+)
 from galaxy.structured_app import BasicSharedApp
 
 
 class ApiKeyManager:
     def __init__(self, app: BasicSharedApp):
         self.app = app
 
-    def create_api_key(self, user) -> str:
+    def get_api_keys(self, user: User) -> List[APIKeys]:
+        return user.api_keys
```

Comment:
That is so weird, users should only have a single active API key.

Reply:
Now the inactive ones will be marked as deleted :)